dist/images/ovnkube.sh

```diff
@@ -534,6 +534,8 @@ create_ovnkube_db_ep () {
   # create a new endpoint for the headless onvkube-db service without selectors
   # using the current host has the endpoint IP
   ovn_db_host=$(getent ahosts $(hostname) | head -1 | awk '{ print $1 }')
+  ovn-nbctl set-connection ptcp:6641:${ovn_db_host} -- set connection . inactivity_probe=0
+  ovn-sbctl set-connection ptcp:6642:${ovn_db_host} -- set connection . inactivity_probe=0
   kubectl --server=${K8S_APISERVER} --token=${k8s_token} --certificate-authority=${K8S_CACERT} create -f - << EOF
```

Comment:
I think Lines 537, 538 seems to be out of place in `create_ovnkube_db_ep()` as the function name and those two lines don't match. Also, when this function gets called from sb-ovsdb() you cannot guarantee that the OVN NB DB is up and running. So, I would move the `getent and set-connection` calls to the respective functions nb-ovsdb() and sb-ovsdb().